README.md

```diff
@@ -15,4 +15,4 @@ When downloading directly from GitHub (by cloning or downloading as zip) you wil
 
 ## About Us
 
-This iTop module development is sponsored, led and supported by [Combodo](https://www.combodo.com).
+This iTop module development is sponsored, led, and supported by [Combodo](https://www.combodo.com).
```

Comment:
Hello @dariocodes , thanks for the PR!

I agree for the first correction, but are you sure about this one?

Reply:
Hi @Molkobain, No problem!

I believe the comma should be added, and I checked some English grammar rules to be sure.

The reason for this is that the sentence uses a series of three items (sponsored, led, and supported) that are separated by commas to show that they are all part of the same list. This is called a serial or Oxford comma, and it is commonly used in English to avoid ambiguity or confusion.

In the absence of the final comma after "led," the sentence could be interpreted as implying that "supported by Combodo" refers to both "sponsored" and "led," rather than just "supported." By adding the final comma, the sentence is made clear that each of the three items in the list is independent and distinct.

Reply:
Thanks for the explanation, TIL!
Yes you can do it on the iTop repo as well